app/models/work.rb

```diff
@@ -0,0 +1,33 @@
+class Work < ApplicationRecord
+    has_many :votes
+    validates :category, presence: true
+    validates :title, presence: true
+
+    def self.media_spotlight
+        Work.all.order(vote_count: :desc).first
+    end
+
+    def self.top_ten_books
```

Comment:
Consider using a single method for all categories that takes the category as a parameter to DRY up your code:`self.top_ten(category)`. Then you can use the `.find` method to get all works from that category, and sort them.